packages/expo-camera/src/ExponentCamera.web.tsx

```diff
@@ -95,44 +101,60 @@ export default class ExponentCamera extends React.Component<NativeProps> {
     if (!ref) {
       this.video = null;
       if (this.camera) {
-        this.camera.unmount();
+        this.camera.stopAsync();
         this.camera = undefined;
       }
       return;
     }
     this.video = findNodeHandle(ref);
+
+    (this.video as any).webkitPlaysinline = true;
+
     this.camera = new CameraModule(ref);
     this.camera.onCameraReady = this.onCameraReady;
     this.camera.onMountError = this.onMountError;
     this._updateCameraProps(this.props);
   };
 
   render() {
-    const transform = this.state.type === CameraManager.Type.front ? 'rotateY(180deg)' : 'none';
-    const style: CSSProperties = {
-      position: 'absolute',
-      top: 0,
-      left: 0,
-      right: 0,
-      bottom: 0,
-      width: '100%',
-      height: '100%',
-      objectFit: 'cover',
-      transform,
+    const { pointerEvents } = this.props;
+
+    // TODO: Bacon: Create a universal prop, on native the microphone is only used when recording videos. 
+    // Because we don't support recording video in the browser we don't need the user to give microphone permissions.
+    const isMuted = true;
+
+    const isFrontFacingCamera = this.state.type === CameraManager.Type.front;
+    const style = {
+      // Flip the camera
+      transform: isFrontFacingCamera ? [{ scaleX: -1 }] : undefined,
     };
 
     return (
-      <View style={[styles.videoWrapper, this.props.style]}>
-        <video ref={this._setRef} style={style} autoPlay playsInline />
+      <View pointerEvents="box-none" style={[styles.videoWrapper, this.props.style]}>
+        <Video 
+          autoPlay 
+          playsInline 
+          muted={isMuted} 
+          pointerEvents={pointerEvents} 
+          ref={this._setRef} 
+          style={[StyleSheet.absoluteFill, styles.video, style]} 
```

Comment:
1. `StyleSheet.absoluteFillObject`
2. Because in RNWeb `absoluteFill` is already an atomic style, if I merge the `absoluteFillObject` then it's just creating a new larger style.